src/plugins/mtbank/api.js

```diff
@@ -4,11 +4,23 @@ import * as _ from 'lodash'
 const baseUrl = 'https://mybank.by/api/v1/'
 
 async function fetchJson (url, options, predicate = () => true, error = (message) => console.assert(false, message)) {
-  const response = await network.fetchJson(baseUrl + url, options || {})
+  options = _.defaultsDeep(
+    options,
+    {
+      sanitizeRequestLog: { headers: { Cookie: true } },
+      sanitizeResponseLog: { headers: { 'set-cookie': true } }
+    }
+  )
+  const response = await network.fetchJson(baseUrl + url, options)
   if (predicate) {
     validateResponse(response, response => predicate(response), error)
   }
 
+  if (!response.body.success && response.body.error && response.body.error.description) {
+    if (response.body.error.description.indexOf('Неверный пароль') >= 0) { throw new InvalidPreferencesError('Ответ банка: ' + response.body.error.description + response.body.error.lockedTime) }
```

Comment:
you can initialize a new variable for the error and use it for InvalidPreferencesError and TemporaryError exceptions

Reply:
Thnx. Fixed :)